src/OpenTelemetry.Exporter.Zipkin/ZipkinExporter.cs

```diff
@@ -57,33 +57,30 @@ public ZipkinExporter(ZipkinExporterOptions options, HttpClient client = null)
         internal ZipkinEndpoint LocalEndpoint { get; }
 
         /// <inheritdoc/>
-        public override async Task<ExportResult> ExportAsync(IEnumerable<Activity> batchActivity, CancellationToken cancellationToken)
+        public override ExportResultSync Export(in Batch<Activity> batch)
         {
             try
             {
-                await this.SendBatchActivityAsync(batchActivity, cancellationToken).ConfigureAwait(false);
+                // take a snapshot of the batch
+                var activities = new List<Activity>();
```

Comment:
A lot of work to avoid allocations just to allocate 🤣 What's the plan here?

Reply:
The plan is to change the serializer to take the zero-alloc foreach instead of IEnumerator.
For now, the immediate goal is to migrate/clean up so we have a stable API for Beta-2, and do micro optimization in separate PRs.

Reply:
Sounds good I figured you would have a plan for it.